vis.c

```diff
@@ -314,11 +314,10 @@ static void window_draw_cursor_matching(Win *win, Selection *cur) {
 static void window_draw_cursor(Win *win, Selection *cur) {
 	if (win->vis->win != win)
 		return;
-	Line *line = view_cursors_line_get(cur);
-	int col = view_cursors_cell_get(cur);
-	if (!line || col == -1)
+	Line *line = cur->line;
+	if (!line)
```

Comment:
Nope, the `col == -1` was only there because `view_cursors_cell_get()` would return `-1` when `cur->line == NULL`.  It was just hidden behind the redirection.